packages/AI/Vectors/Sync/src/models/entityVectorSync.ts

```diff
@@ -104,6 +104,9 @@ export class EntityVectorSyncer extends VectorBase {
       contextUser: super.CurrentUser,
       workerContext,
     });
+    annotator.on('error', (err) => {      
```

Comment:
We really should rename these to be more descriptive as well, but this is still a big improvement